src/main/java/uk/gov/hmcts/reform/civil/service/DashboardNotificationsParamsMapper.java

```diff
@@ -29,6 +33,10 @@ public Map<String, Object> mapCaseDataToParams(CaseData caseData) {
             params.put("defendantAdmittedAmount", formatAmount(getDefendantAdmittedAmount(caseData)));
         }
 
+        if (nonNull(caseData.getRespondToClaimAdmitPartLRspec())) {
+            params.put("defendantAdmittedAmountPaymentDeadline", caseData.getRespondToClaimAdmitPartLRspec().getWhenWillThisAmountBePaid());
+        }
```

Comment:
This value already exists. Check the next if. Also the value that exists is split in 2 to do the date translations for EN and CY as Dharmendra requested.